driver/src/main.rs

```diff
@@ -202,10 +205,15 @@ fn main() {
     let price_finder = price_finding::create_price_finder(fee, options.solver_type, price_oracle);
 
     // Create the orderbook reader.
-    let orderbook = PaginatedStableXOrderBookReader::new(&contract, options.auction_data_page_size);
-    info!("Orderbook filter: {:?}", options.orderbook_filter);
+    let primary_orderbook =
```

Comment:
Maybe we should have a shadow flag that we can pass in to the run command so we only run the shadow orderbook on one of our machines.